frontend/src/loadPostHogJS.tsx

```diff
@@ -50,6 +50,10 @@ export function loadPostHogJS(): void {
                 _onCapture: (window as any)._cypress_posthog_captures
                     ? (_, event) => (window as any)._cypress_posthog_captures.push(event)
                     : undefined,
+
+                session_recording: {
+                    compress_events: true,
```

Comment:
this turns this on for us
without affecting posthog.com

the js code is out but not running for anyone right now